pages/common/mmls.md

```diff
@@ -0,0 +1,20 @@
+# mmls
+
+> Display the partition layout of a volume system.
+> More information: <https://wiki.sleuthkit.org/index.php?title=Mmls>.
+
+- Display the partition table stored in an image file:
+
+`mmls {{path/to/image_file}}`
+
+- Display the partition table with an additional column for the partition size:
+
+`mmls -B -i {{path/to/image_file}}`
```

Comment:
What does B stand for exactly ? Is there a full version of the flag ? Is there a way we can incorporate the word in the description ?

Reply:
Reading the man page with `debman -p sleuthkit mmls`, it doesn't look like there are long flags :-/